Makefile

```diff
@@ -10,7 +10,6 @@ $(VIRTUALENV)/.installed:
 	$(VIRTUALENV)/bin/pip3 install -r requirements.txt
 	$(VIRTUALENV)/bin/pip3 install -r requirements_test.txt
 	$(VIRTUALENV)/bin/pip3 install -e .
-	$(VIRTUALENV)/bin/pip3 install git+https://github.com/epfml/sent2vec.git # otherwise it fails cause setup.py relies on cython and numpy being installed
```

Comment:
the problem is caused from removing this line. adding it should allow you remove the need for `build/virtuelanv/bin/pip`. this is not a good permanent fix but i am working on a better solution

Reply:
cool. Maybe it makes sense to leave this PR until you've merged your PR and then I'll rebase on it?